__tests__/commands/install/bin-links.js

```diff
@@ -175,6 +175,15 @@ test('can use link protocol to install a package that would not be found via nod
   });
 });
 
+test('empty bin string does not create a link', (): Promise<void> => {
+  return runInstall({binLinks: true}, 'install-empty-bin', async config => {
+    const binScripts = await fs.walk(path.join(config.cwd, 'node_modules', '.bin'));
+    expect(binScripts).toHaveLength(1);
```

Comment:
This should have length 2 on Windows, as two files are created per binary on Windows (one Bash script for linux-like environments like Cygwin, and one `.cmd` Batch script). This is why the appveyor tests are failing.